src/current/v24.2/changefeed-sinks.md

```diff
@@ -495,13 +496,13 @@ The following are considerations when using the webhook sink:
 
 * Only supports HTTPS. Use the [`insecure_tls_skip_verify`]({% link {{ page.version.version }}/create-changefeed.md %}#insecure-tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates.
 * Supports JSON output format. You can use the [`format=csv`]({% link {{ page.version.version }}/create-changefeed.md %}#format) option in combination with [`initial_scan='only'`]({% link {{ page.version.version }}/create-changefeed.md %}#initial-scan) for CSV-formatted messages.
-* There is no concurrency configurability.
```

Comment:
in v24.1 you removed this but also added a bullet about configuring concurrency but that appears to be the only version where that is done.  I don't understand

Reply:
Yes, I was meant to do the same in v24.1. The only differences between the versions are where the cluster settings for the sinks are default or not.